test/integration/fill-survey.spec.js

```diff
@@ -11,7 +11,7 @@ describe('Explore work you could do survey', () => {
   const theType = 'ewycd';
   const theRef = '12345';
 
-  function fetchFirstSurveyFromDB() {
+  function fetchFirstAnswersFromDB() {
```

Comment:
maybe `fetchFirstAnswersObjectFromDb` . this represent list of the survey answers